src/redis/README.md

```diff
@@ -0,0 +1,28 @@
+# VIP Go Redis Helper Library
+
```

Comment:
We should also note that our Redis implementation is based on IORedis (and link to that project).

Reply:
Yeah good call, added in https://github.com/Automattic/vip-go-node/pull/188/commits/63b1e490bd043d920170de5d363689f8442df7c7